integration_tests/kinesis-s3-source/create_source.sql

```diff
@@ -15,7 +21,13 @@ CREATE SOURCE ad_impression (
 CREATE SOURCE ad_click (
     bid_id BIGINT,
     click_timestamp TIMESTAMPTZ
-) WITH (
+)
+INCLUDE key
+INCLUDE partition
+INCLUDE offset
+INCLUDE timestamp
+INCLUDE payload
```

Comment:
No test results need to be updated accordingly?

Reply:
integration tests have limited ability on check results. Let's find another way.